### DISCUSSION TITLE
`.DefaultConfig` question

### DISCUSSION THREAD

**Discussion by Malix-Labs** [disgoorg/disgo#365] [Help] [answered]
Hey,

If we make a custom config, will it still call `.DefaultConfig()` beforehand?

For example, about `httpserver.DefaultConfig()`, the port `:80` is chosen by default,
Will we need to specify it again if we call a custom config only editing the base url?

**Comment by topi314** [comment, marked as answer]
Since you are passing opts, aka functions modifying the config no.

The default config is what gets changed by the passed opts and only that specific thing there is no need to respecify those
